styx-scheduler-service/src/test/java/com/spotify/styx/state/handlers/DockerRunnerHandlerTest.java

```diff
@@ -217,6 +220,21 @@ public void shouldPerformCleanupOnFailedThroughTransitions() {
     verify(dockerRunner).cleanup(workflowInstance, TEST_EXECUTION_ID);
   }
 
+  @Parameters({"SUBMITTED", "RUNNING"})
+  @Test
+  public void shouldPollWhenRunning(State state) {
```

Comment:
`s/shouldPollWhenRunning/shouldPollWhenInState` ?

Reply:
Or perhaps `shouldPollWhenStarted`. I want the test name to be descriptive.